### PR TITLE
✨(edxapp) allow to change pvc media name

### PR DESCRIPTION
**Pull request by lunika**
## Purpose

We want to rename as we want the edxapp pvc media name. For this we
introduced a new variable edxapp_pvc_media_name and fallback on a
default value if not defined.

## Proposal

- [x] allow to change pvc media name

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
+
+- Allow to change pvc media name
```
Comment (jmaupetit) [inline CHANGELOG.md]:
```suggestion
- Allow to change PVC media name
```

**Review by jmaupetit — COMMENTED** [review]
Great feature. Why not adding it to all `edxapp` PVCs?

**Comment by lunika** [comment]
> Great feature. Why not adding it to all `edxapp` PVCs?

Because we are KISS and YAGNI addict.

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**